lib/LocalCluster.js

```diff
@@ -27,6 +27,62 @@ var LocalCentroid = require(PATH + 'LocalCentroid');
 var constants = require(PATH + 'constants');
 
 
+function countItemsMatches(text, items) {
+  /**
+   * Check if a word is contained or not
+   * @param {string} text Input text (single word)
+   * @param {string} items to match
+   */
+   for (indx = 0; indx < items.length; indx++) {
+       if (text === items[indx][0]) {
+         return 1;
+       }
+   }
+   return 0;
+}
+
+
+function itemMatches(text, summaryItems) {
+  /**
+   * Computes item matches depending on the chosen items analysis options
```

Comment:
Sorry, as @joseribes pointed out, the items field has only text separated by a user-given separator or regular expression, so either you have a list of text tokens or you tokenize using the user-given information.